cli/info.rs

```diff
@@ -221,6 +226,92 @@ impl FileInfoDepTree {
       deps,
     }
   }
+
+  /// Flattens dependencies
+  ///
+  /// Returns flat graph structure with dependencies list per module file
+  fn flatten_to_graph(&self) -> HashMap<String, FileInfoVertex> {
+    let mut flat_graph = HashMap::new();
+    dig_out_nested_deps(&self, &mut flat_graph);
+    flat_graph
+  }
+}
```

Comment:
@bartOssh why not use `ModuleGraph` which is already a `HashMap` to extract info? Should be quite easier than traversing the tree.

Reply:
Additionally, keys should be sorted to preserve order of files in different tests.

Reply:
Thanks. I missed that one!